src/main/java/eu/bitwalker/useragentutils/OperatingSystem.java

```diff
@@ -122,7 +122,15 @@
 		iOS6_IPAD(	Manufacturer.APPLE, OperatingSystem.MAC_OS_X_IPAD, 51, "iOS 6 (iPad)", new String[] { "OS 6" },  null, DeviceType.TABLET, null ), // before Mac OS X
 		MAC_OS_X_IPHONE(Manufacturer.APPLE, OperatingSystem.IOS, 40, "Mac OS X (iPhone)", new String[] { "iPhone" },  null, DeviceType.MOBILE, null ), // before Mac OS X
 		MAC_OS_X_IPOD(	Manufacturer.APPLE, OperatingSystem.IOS, 30, "Mac OS X (iPod)", new String[] { "iPod" },  null, DeviceType.MOBILE, null ), // before Mac OS X
-	
+	iOS8_4(	Manufacturer.APPLE, null, 101, "iOS 8.4", new String[] { "CFNetwork/711.4.6" },  null, DeviceType.MOBILE, null ), // use CFNetwork
```

Comment:
Those should be children of iOS. 
So, if I understand it correctly, we can't tell if it is a phone or iPad. We only know that is is an iOS device.
So there is a certain error rate, qualifying these as DeviceType.MOBILE.